numpy/_core/src/multiarray/ctors.c

```diff
@@ -2466,6 +2466,7 @@ PyArray_FromArrayAttr_int(
 
     if (new == NULL) {
         if (npy_ma_str_array_err_msg_substr == NULL) {
+            Py_DECREF(array_meth);
```

Comment:
Do `args` and `kwargs` need to be DECREF'd here?

Reply:
Thanks for spotting